fe/fe-core/src/main/java/com/starrocks/sql/ast/expression/RowPositionDescriptor.java

```diff
@@ -0,0 +1,81 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.starrocks.analysis;
+
+import com.google.common.base.Preconditions;
+import com.starrocks.planner.SlotId;
+import com.starrocks.thrift.TRowPositionDescriptor;
+import com.starrocks.thrift.TRowPositionType;
+
+import java.util.List;
+
+// describe how to find a row in specific table, only used by global late materialization
+public class RowPositionDescriptor {
+    public enum Type {
+        ICEBERG_V3
+    }
+    private Type type;
+    private SlotId rowSourceSlot;
+    private List<SlotId> fetchRefSlots;
+    private List<SlotId> lookupRefSlots;
```

Comment:
why fetchRefSlots and lookupRefSlots are different?  scan_range_id is not required actually?